experiments/AMIP/moist_mpi_earth/slab/slab_utils.jl

```diff
@@ -0,0 +1,26 @@
+struct SlabSimulation{P, Y, D, I}
+    params::P
+    Y_init::Y
+    domain::D
+    integrator::I
+end
+
+get_slab_energy(slab_sim, T_sfc) = slab_sim.params.ρ .* slab_sim.params.c .* T_sfc .* slab_sim.params.h
+
+function get_land_energy(slab_sim::SlabSimulation, T_sfc)
+    return get_slab_energy(slab_sim, T_sfc)
+end
+
+function get_land_temp(slab_sim)
+    return slab_sim.integrator.u.T_sfc
+end
+
+function get_land_roughness(slab_sim::SlabSimulation)
+    return slab_sim.integrator.p.params.z0m, slab_sim.integrator.p.params.z0b
+end
+
+land_albedo(slab_sim::SlabSimulation) = slab_sim.params.α
+
+function get_land_q(slab_sim::SlabSimulation, atmos_sim, T_land, ρ_sfc)
+    return TD.q_vap_saturation_generic.(atmos_sim.integrator.p.params, T_land, ρ_sfc, TD.Liquid())
+end
```

Comment:
```suggestion

```
We can remove these as they are defined for the bucket now in bucket_init

Reply:
do we want to preserve being able to use a slab land instead of the bucket? it's a minor difference - the bucket energy part is a slab, but it is based on a different repo, etc. These methods would be necessary if we want to retain both.